kubeconfig.go

```diff
@@ -40,18 +39,7 @@ func New(config Config) (*KubeConfig, error) {
 
 // NewRESTConfigForApp returns a Kubernetes REST config for the cluster
 // configured in the kubeconfig section of the app CR.
-func (k *KubeConfig) NewRESTConfigForApp(ctx context.Context, app v1alpha1.App) (*rest.Config, error) {
-	if inCluster(app) {
```

Comment:
I will check this `inCluster` from operator itself.

Reply:
OK cool I think that's fine. 👍